app/controllers/questions_controller.rb

```diff
@@ -10,28 +10,63 @@ def index
     @question_limitations = ENV["limit_question_index_to_users_questions_only"]
   end
 
+  def show
+    @questions = get_question_list()
+    @question_id = params[:id]
+    @question = @questions[(params[:id]).to_i - 1]["text"]
+
+    logger.debug "QUESTION's type in question's show: #{@questions[(params[:id]).to_i - 1].class}"
+
+    @answers = @questions[(params[:id]).to_i - 1]["answers"]
+    @answer = Answer.new
+  end
+
   def new
     @question = Question.new
     @question.answers.build
+    logger.debug "@question from new is #{@question}"
   end
 
   def create
     question = Question.new(params.require(:question)
       .permit(:text,
         answers_attributes:[:text]))
+
+    logger.debug "QUESTION's type in question's create: #{question.class}"
+
     question.user_id = current_user.id
     question.post_question(question)
+
+    logger.debug "@QUESTION's type in question's destroy: #{@question.class}"
+
     redirect_to new_question_path, notice: "Your question has been submitted! Enter another if you would like."
   end
 
-  def show
+  # def show
```

Comment:
Please remove the commented-out code (it's still in the git history if you need it later).